ramp-database/ramp_database/testing.py

```diff
@@ -47,18 +48,35 @@ def create_test_db(database_config, ramp_config):
         The configuration file containing the database information.
     ramp_config : dict
         The configuration file containing the information about a RAMP event.
+
+    Returns
+    -------
+    deployment_dir : str
+        The deployment directory for the RAMP components (kits, data, etc.).
     """
     database_config = database_config['sqlalchemy']
     # we can automatically setup the database from the config file used for the
     # tests.
     ramp_config = generate_ramp_config(ramp_config)
-    shutil.rmtree(ramp_config['deployment_dir'], ignore_errors=True)
+
+    # FIXME: we are recreating the deployment directory but it should be
+    # replaced by an temporary creation of folder.
+    match = SequenceMatcher(
+        None, ramp_config['ramp_kit_dir'], ramp_config['ramp_submissions_dir']
```

Comment:
you can use `os.path.commonpath` here (eg between ramp_kit_dir and ramp_data_dir), that would already be much simpler I think. 
But we could also just hard code it, as for the tests we know what the it is (or which part of the path it is, eg do a os.path.split twice on ramp_kit_dir)

Reply:
oh. i did not search properly. Thanks